twitter/selenium_twitter_scraper.py

```diff
@@ -241,7 +260,8 @@ def store_tweet_data_payload(searchQuery, dataPayload):
 driver = Chrome(service=Service(ChromeDriverManager().install()), options=options) #Firefox: "Firefox" | Edge: "options = EdgeOptions(); options.use_chromium = True; driver = Edge(options=options)"
 logger.debug("--- Created Chrome driver ---  ")
 
-driver.implicitly_wait(20) # Better Practice to use this than time.sleep() (Unlike 'time.sleep()', 'driver.implicitly_wait()' is NOT a FIXED wait time) [Gives more time to load webpage to find elements]
```

Comment:
Bring this back! From my experience, should reduce scroll margin error received

Reply:
yes daddy